### PR TITLE
GEOPY-1265

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1265 - Log bounds applied to twice on joint inversion**

### CODE REVIEW

File: geoapps/inversion/components/models.py [hunk 1]
```diff
@@ -114,71 +114,73 @@ def active_cells(self, active_cells):
...
         if mstart is not None and self.is_sigma:
-            mstart = mstart.copy()
             mstart = np.log(mstart)
 
         return mstart
 
     @property
-    def reference(self):
+    def reference(self) -> np.ndarray | None:
         mref = self._reference.model
```
Comment (benk-mira) [inline geoapps/inversion/components/models.py:130]: The logic in here is complicated.  Is there a way to combine the `if mref is None` and `if self.is_sigma & (all(mref == 0))`  and can we stick with one name for reference model?

**Review by benk-mira — CHANGES_REQUESTED** [review]
Not clear how the diff solves the described issue..  I'll check during test.

**Comment by benk-mira** [comment]
Whoops, I had the wrong issue in mind when I submitted my review!  ignore the comment about the diff not making sense!

File: geoapps/inversion/components/models.py (hunk 1 above)
Comment (domfournier) [reply geoapps/inversion/components/models.py:130]: You got it.

**Review by benk-mira — APPROVED** [review]

**Pull request merged**